notifications/tests/sample_notifications/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from notifications.base.models import AbstractNotification
+
+
+class DetailsModel(models.Model):
+    details = models.CharField(max_length=64, blank=True, null=True)
```

Comment:
why don't you define this field directly on the model below in order to improve readability?